router/router.go

```diff
@@ -78,34 +78,39 @@ func HealthCheck(c *gin.Context) {
 	c.Writer.WriteHeader(http.StatusOK)
 }
 
-//CallbackRequest will be triggered by the provider automatically after the login
+// CallbackRequest will be triggered by the provider automatically after the login
 func CallbackRequest(c *gin.Context) {
 	var user *models.UserCredentials
 	var err error
 	urlString := types.PortalURL + "/login?"
 	state := c.Query("state")
-
 	switch state {
 	case types.GithubState:
-		{
-			user, err = providers.GetGithubUser(c)
-			if err != nil {
-				log.Errorln("Error getting user from github", err)
-				c.JSON(http.StatusInternalServerError, gin.H{
-					"error": err.Error(),
-				})
-				c.Redirect(http.StatusFound, urlString)
-				return
-			}
-		}
-	default:
-		{
+		user, err = providers.GetGithubUser(c)
+		if err != nil {
+			log.Errorln("Error getting user from github", err)
 			c.JSON(http.StatusInternalServerError, gin.H{
-				"error": "state Invalid",
+				"error": err.Error(),
 			})
 			c.Redirect(http.StatusFound, urlString)
 			return
 		}
+	case types.GoogleState:
+		user, err = providers.GetGoogleUser(c)
+		if err != nil {
+			log.Errorln("Error getting user from google", err)
+			c.JSON(http.StatusInternalServerError, gin.H{
+				"error": err.Error(),
+			})
+			log.Info(urlString)
+			c.Redirect(http.StatusFound, urlString)
```

Comment:
return missing? Seeing it for the other 2 cases.

Reply:
I've added the return statements in the cases